apps/researcher/src/lib/community.ts

```diff
@@ -28,8 +31,58 @@ export async function getMemberships(
   });
 }
 
-export function getAllCommunities(): Promise<Community[]> {
-  return clerkClient.organizations.getOrganizationList();
+export enum CommunitySortBy {
```

Comment:
Perhaps rename `CommunitySortBy` to `SortBy` (since the file (`communities`) already makes the scope clear)?